x-pack/plugins/gis/public/shared/layers/heatmap_layer.js

```diff
@@ -130,19 +131,24 @@ export class HeatmapLayer extends AbstractLayer {
     const metricPropertyKey = this._getPropKeyOfSelectedMetric();
     const updateDueToMetricChange = !_.isEqual(dataMeta.metric, metricPropertyKey);
 
+    const updateDueToResolutionChange = !_.isEqual(dataFilters.geohashResolution, this._source.getGridResolution());
```

Comment:
I don't think this check is needed. Won't `isSamePrecision` provide the same check?

Reply:
agreed